### DISCUSSION TITLE
Full screen editor?

### DISCUSSION THREAD

**Discussion by mord0d** [xenocrat/chyrp-lite#242] [Ideas]
It would be *great* to be able to open an editor in full screen mode. Like [this](https://codemirror.net/5/demo/fullscreen.html) but with a button to be able to switch mode on a mobile device which doesn't have physical keyboard.

**Comment by xenocrat** [comment]
Thanks for the example. I'll consider it.

**Comment by xenocrat** [comment]
I've just added [this commit](https://github.com/xenocrat/chyrp-lite/commit/a1181f2eebebbb6ef0d3d61c556dd5040c76756a) to adapt text areas for maximum visible text lines on mobile screens. This isn't exactly what you requested (and I might in the future still implement that feature) but it should go some way to improving the writing experience on small screens.
